File: Programmers/Lv.0/로그인 성공?/120883.js
function solution(id_pw, db) {
    let answer = "fail";
    
    const [id, pw] = id_pw;
    db.forEach(arr => {
        if (id === arr[0]) {
            if (pw === arr[1]) {
                answer = "login";
                return;
            }
            
            answer = "wrong pw";
        }
    });
    
    return answer;
}

File: Programmers/Lv.0/로그인 성공?/120883.java
class Solution {
    public String solution(String[] id_pw, String[][] db) {
        String answer = "fail";
        
        String id = id_pw[0];
        String pw = id_pw[1];
        for (String[] arr : db) {
            if (id.equals(arr[0])) {                
                if (pw.equals(arr[1])) {
                    answer = "login";
                    break;
                }
                
                answer = "wrong pw";
            }
        }
        
        return answer;
    }
}
